guide/dynamics/dynamics-floquet.rst

```diff
@@ -104,96 +104,98 @@ Consider for example the case of a strongly driven two-level atom, described by
+   >>> f_modes_0 # doctest: +NORMALIZE_WHITESPACE
+   [Quantum object: dims = [[2], [1]], shape = (2, 1), type = ket
+   Qobj data =
+   [[ 0.72964231+0.j      ]
+    [-0.39993746+0.554682j]],
+   Quantum object: dims = [[2], [1]], shape = (2, 1), type = ket
+   Qobj data =
+   [[0.39993746+0.554682j]
+    [0.72964231+0.j      ]]]
 
 For some problems interesting observations can be draw from the quasienergy levels alone. Consider for example the quasienergies for the driven two-level system introduced above as a function of the driving amplitude, calculated and plotted in the following example. For certain driving amplitudes the quasienergy levels cross. Since the the quasienergies can be associated with the time-scale of the long-term dynamics due that the driving, degenerate quasienergies indicates a "freezing" of the dynamics (sometimes known as coherent destruction of tunneling).
 
-.. ipython::
-
-   In [1]: delta = 0.2 * 2*np.pi; eps0  = 0.0 * 2*np.pi
-
-   In [1]: omega = 1.0 * 2*np.pi; A_vec = np.linspace(0, 10, 100) * omega;
-
-   In [1]: T = (2*pi)/omega
-
-   In [1]: tlist  = np.linspace(0.0, 10 * T, 101)
-
-   In [1]: psi0   = basis(2,0)
-
-   In [1]: q_energies = np.zeros((len(A_vec), 2))
-
-   In [1]: H0 = delta/2.0 * sigmaz() - eps0/2.0 * sigmax()
-
-   In [1]: args = {'w': omega}
-
-   In [1]: for idx, A in enumerate(A_vec):
-      ...:     H1 = A/2.0 * sigmax()
-      ...:     H = [H0, [H1, lambda t, args: sin(args['w']*t)]]  
-      ...:     f_modes, f_energies = floquet_modes(H, T, args, True)
-      ...:     q_energies[idx,:] = f_energies
-
-   In [1]: figure()
-
-   In [1]: plot(A_vec/omega, q_energies[:,0] / delta, 'b', A_vec/omega, q_energies[:,1] / delta, 'r')
-
-   In [1]: xlabel(r'$A/\omega$')
-
-   In [1]: ylabel(r'Quasienergy / $\Delta$')
-
-   In [1]: title(r'Floquet quasienergies')
-
-   @savefig guide-floquet1.png width=5.0in align=center
-   In [1]: show()
+.. plot::
+   :context:
+
+   >>> delta = 0.2 * 2*np.pi
+   >>> eps0  = 0.0 * 2*np.pi
+   >>> omega = 1.0 * 2*np.pi
+   >>> A_vec = np.linspace(0, 10, 100) * omega
+   >>> T = (2*pi)/omega
+   >>> tlist = np.linspace(0.0, 10 * T, 101)
+   >>> spsi0 = basis(2,0)
+   >>> q_energies = np.zeros((len(A_vec), 2))
+   >>> H0 = delta/2.0 * sigmaz() - eps0/2.0 * sigmax()
+   >>> args = {'w': omega}
+   >>> for idx, A in enumerate(A_vec): # doctest: +SKIP
+   >>>   H1 = A/2.0 * sigmax() # doctest: +SKIP
+   >>>   H = [H0, [H1, lambda t, args: sin(args['w']*t)]] # doctest: +SKIP
+   >>>   f_modes, f_energies = floquet_modes(H, T, args, True) # doctest: +SKIP
+   >>>   q_energies[idx,:] = f_energies # doctest: +SKIP
+   >>> figure() # doctest: +SKIP
+   >>> plot(A_vec/omega, q_energies[:,0] / delta, 'b', A_vec/omega, q_energies[:,1] / delta, 'r') # doctest: +SKIP
+   >>> xlabel(r'$A/\omega$') # doctest: +SKIP
+   >>> ylabel(r'Quasienergy / $\Delta$') # doctest: +SKIP
+   >>> title(r'Floquet quasienergies') # doctest: +SKIP
+   >>> show() # doctest: +SKIP
```

Comment:
Is it appearing for you? It is not supposed to appear and isn't for me right now.

Reply:
![image](https://user-images.githubusercontent.com/12125783/90321346-081dd180-df49-11ea-988d-18d58c7cc853.png)

It rendered like this for me. It is not for you? That's weird.

BTW, why skipping the test here?

Reply:
Yeah, they arent rendering for me when in html mode atleast. 

I am skipping the tests because they seem to be glitching a bit (the plots are not being tested anyway given we dont have any meaninful output values for them). They will be "tested" when doing *make html* anyway.